package.json

```diff
@@ -24,7 +24,7 @@
     "intl": "1.2.5",
     "jail-monkey": "2.3.1",
     "jsc-android": "241213.2.0",
-    "mattermost-redux": "github:mattermost/mattermost-redux#af7a071de69e778a633ab182367568c21977f6b0",
+    "mattermost-redux": "github:mattermost/mattermost-redux#a0e7a1cbedb2ec051e0d2411fd47ee19bb49d709",
```

Comment:
Does this work?

```
"mattermost-redux": "github:mattermost/mattermost-redux#release-5.20"
```
Seems less error-prone to match on tags rather than commit SHAs?

Reply:
We always point to a commit to avoid unnecessary changes if the branch has a new commit, also it avoids updating package-lock constantly